text/0054-remove-heap-pages.md

```diff
@@ -0,0 +1,91 @@
+# RFC-0054: Remove the concept of "heap pages" from the client
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2023-11-24                                                                                  |
+| **Description** | Remove the concept of heap pages from the client and move it to the runtime.                |
+| **Authors**     | Pierre Krieger                                                                              |
+
+## Summary
+
+Rather that enforce a limit to the total memory consumption on the client side by loading the value at `:heappages`, enforce that limit on the runtime side.
```

Comment:
```suggestion
Rather than enforce a limit to the total memory consumption on the client side by loading the value at `:heappages`, enforce that limit on the runtime side.
```